pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -339,6 +341,20 @@ func (v *vpnSeedServer) Deploy(ctx context.Context) error {
 									Value: v.podNetwork,
 								},
 							},
+							ReadinessProbe: &corev1.Probe{
```

Comment:
It would be nice to have a some initial delay via `initialDelaySeconds`.
Also, the default timeout is 1 second, you can increase it via `timeoutSeconds`.

Reply:
@vpnachev I am not sure what you are saying here. Do you want explicit values instead of the defaults? When I checked the defaults they looked fine.

Reply:
Let's give it a try with the default values.